ghost/webmentions/lib/MentionSendingService.js

```diff
@@ -1,5 +1,6 @@
 const errors = require('@tryghost/errors');
 const logging = require('@tryghost/logging');
+const settingsCache = require('../../core/core/shared/settings-cache');
```

Comment:
A different possibility I'm only thinking about now is to just change the implementation of `isEnabled` in the wrapper service 😊

Reply:
Both approaches work for me 👌

Reply:
I was also thinking that re: `IsEnabled`. Taking a look back at this after the weekend I believe I see why we can't import from core (given that these are packages, not part of core). 

We should just be able to import from core like this within `core/core/server/services/mentions/service.js`, right? Then we could either pass into the constructor or encapsulate it in `isEnabled` but I think I like the idea of tying it in to `isEnabled`. Plus eventually this won't be a lab, and it'll be easier to retire that way.

Reply:
Yup, you are right 😊 The idea is that we pass or use the core classes and files in the wrapper service so other packages have those things available via dependency injection.

Go ahead with adding it in isEnabled 😊 Nice thing about that is that the package doesn't need any changes or new tests. It could use a E2E test (= in core) or something similar to make sure it actually work